### ISSUE TITLE
Spotify tools: Bad URI / active device guidance gap in playback/queue tools

### ISSUE THREAD

**Issue by rodrigogs** [NousResearch/hermes-agent#56044]
Spotify tools raise Bad URI / missing active device / malformed URI errors on common inputs.

- Failure when passing raw IDs or URIs that the current normalizer does not handle.
- Failure when calling playback/queue actions with no active device.

Related PR: #56043

**Comment by rodrigogs** [comment]
Aberto PR associado: https://github.com/NousResearch/hermes-agent/pull/56043

**Comment by GottZ** [comment]
> *This was generated by AI during triage.*

#### Summary

One open PR addresses the reported Spotify URI-normalization and missing-active-device failures. Its diff adds canonicalization for URLs, native URIs, and typed raw IDs, checks for an actually active device, preserves playback batch normalization, and adds regression coverage; the recorded Verify verdict remains unverified.

#### Related pull requests

- #56043 `unverified` — (+152/-4) — unverified: adds Spotify URI/URL/raw-ID coercion, promotes queue IDs to track URIs, distinguishes active devices from merely listed devices, returns actionable no-active-device errors, and adds normalization and queue/device regression tests. The [PR review:COMMENTED] contributor review (keep_open; salvageability=medium) identified correctness gaps; the later visible diff addresses the listed batch-normalizer, active-device, and queue raw-ID points, but no Verify verdict is recorded.

#### Suggested consolidation

keep open with a salvage path: retain the focused URI and active-device changes in #56043, obtain a fresh Verify verdict for the current revision, and confirm handler-level coverage for empty playback input, inactive-only device lists, lookup failures, and queue raw-ID coercion. Do not merge or close it as a duplicate; the contributor review explicitly says keep_open, and the PR is the only PR listed for this issue.

#### Complex graph

```mermaid
flowchart LR
    classDef open fill:#dbeafe,stroke:#1d4ed8,color:#1e3a8a
    classDef merged fill:#dcfce7,stroke:#15803d,color:#14532d
    classDef closed fill:#e5e7eb,stroke:#6b7280,color:#1f2937
    classDef unverified fill:#f3f4f6,stroke:#9ca3af,color:#374151
    classDef best stroke-width:3px,stroke:#b45309
    classDef target stroke-width:3px,stroke:#4338ca
    I56044(["issue #56044 (open)"])
    P56043["PR #56043 (open)"]
    P56043 -.->|unverified| I56044
    class I56044 open
    class I56044 target
    class P56043 open
    click I56044 "https://github.com/NousResearch/hermes-agent/issues/56044"
    click P56043 "https://github.com/NousResearch/hermes-agent/pull/56043"
```
*Graph: solid arrow = fixes / best fix, dashed arrow = partial or unverified (see edge label); boxed group = PRs duplicating each other; amber border = best fix; indigo border = target; gray node = closed (state tag in the node label).*

_Cross-PR triage: Reviewed 1 pull request and 1 issue in this complex. Each diff was read against this issue; Assessment working set: 10 kB of PR diffs, 1 kB of issue/PR text, 5 kB of discussion (5 comments), 0 verify verdicts. verdicts reflect diff content, not PR titles. Part of an automated triage batch._
